ajax/libs/just-animate/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "just-animate",
+  "filename": "just-animate-animations.min.js",
+  "author": "Christopher Wallis <christopher.j.wallis@gmail.com>",
+  "version": "0.6.0",
+  "description": "Web Animation library",
+  "keywords": [
+    "webanimation",
+    "animate",
+    "animatecss",
+    "javascript",
+    "animation",
+    "timeline",
+    "sequence"
+  ],
+  "license": "MIT",
+  "homepage": "https://just-animate.github.io/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/just-animate/just-animate"
+  },
+  "npmName": "just-animate",
+  "npmFileMap": [
+    {
+      "basePath": "browser",
+      "files": [
+        "just-animate*-!(*.ts)"
```

Comment:
@kennynaoh I think the `*` concatenate after `animate` is meaningless. Let's use `just-animate-!(*.ts)`